halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/providers/dockerRegistry/DockerRegistryAccountValidator.java

```diff
@@ -53,9 +56,9 @@ public void validate(ConfigProblemSetBuilder p, DockerRegistryAccount n) {
     }
 
     if (passwordProvided) {
-      resolvedPassword = password;
+      resolvedPassword = secretSessionManager.decrypt(password);
     } else if (passwordFileProvided) {
-      resolvedPassword = ValidatingFileReader.contents(p, passwordFile);
+      resolvedPassword = secretSessionManager.decrypt(ValidatingFileReader.contents(p, passwordFile));
```

Comment:
The value of `passwordFile` may be an encrypted secret. `secretSessionManager.decryptFile` should be called to get the localPasswordFile (it returns the given value if it's not an encrypted secret, so it's ok to call each time). We can then just call `ValidatingFileReader.contents(p, passwordFile)` with the new value for localPasswordFile to get the resolvedPassword